fetch-utils.js

```diff
@@ -18,19 +18,33 @@ export async function createTodo(todo) {
 
 export async function deleteAllTodos() {
     // delete all todos for this user in supabase
-
+    const response = await client
+        .from('todos')
+        .delete()
+        .match({ user_id: client.auth.user().id, });
     return checkError(response);
 }
 
 export async function getTodos() {
     // get all todos for this user from supabase
+    const response = await client
+        .from('todos')
+        .select()
+        .order('complete')
```

Comment:
Great work Sydnie! FYI ordering by Id will allow for the list to not change order when a user completes the todo. :)